report/assets/styles.css

```diff
@@ -312,31 +312,31 @@
   color: var(--report-text-color);
 }
 
-.lh-root :focus {
```

Comment:
I don't love the use of `lh-vars` for these purposes here :/

`lh-scope` doesn't exactly feel right either. IIRC, we used to put all the `lh-vars` on `lh-root` but then one environment didn't end up having an `lh-root` because it split everything out and we moved it onto `lh-vars` instead. Will that same problem happen again with `lh-scope`? Should we just keep both of these? That doesn't actually seem like the worst outcome.

Reply:
I just don't want to use 'root' where there are multiple of them.  And yeah in the PSI case we have the multiple 'partials' and they all need these classes.    I was thinking 'scope' felt good as you can have multiple. and then these L305-325 styles feel a little better on a "scope" than in a var-only thing.


lemme try finding a new home for these styles....

Reply:
okay i was able to place them more specifically: 98cf7d63bdc17f891d8f3d72124a699e7e74a633

lg?

Reply:
I know we deleted `lh-container`, but that seems like a good name for the root/vars.

Reply:
i just pushed one last change which removes the one place i had been applying actual styles with `lh-vars`. (this went against the spirit of the name, yeah)

so now we're back to lh-vars only being for providing the vars. :)  

maybe its my recency bias, but when i see `lh-container` i feel like there should be only one, but.. its true its not as explicit as `root`.